docker/bigtop-puppet/build.sh

```diff
@@ -23,7 +23,6 @@ if [ $# != 1 ]; then
   echo "Usage: build.sh <PREIX-OS-VERSION>"
   echo
   echo "Example: build.sh trunk-centos-7"
-  echo "openeuler: Example: build.sh trunk-openeuler/openeuler-22.03"
```

Comment:
This line (and the below lines L67-71) was added by you in your [previous commit in the same PR](https://github.com/apache/bigtop/pull/1051/commits/da5e75519d2110abfcb27fb118eb43affcb14720).
That seems unnecessary for keeping the history clean.

Would you please squash the two commits?

Reply:
The example output information deleted.

Reply:
No. I mean you submitted two commits which modified the same file, but one added several lines, the other deleted them. Why don't you put these changes into one commit?

This adds: https://github.com/apache/bigtop/pull/1051/commits/da5e75519d2110abfcb27fb118eb43affcb14720 
This deletes: https://github.com/apache/bigtop/pull/1051/commits/a46e50e432b830907deca32bfdbaf63dabbb1ee7

Reply:
These modifications were introduced to resolve different issues during the validation process.